### PR TITLE
Env doctrine/orm version

### PR DESCRIPTION
**Pull request by gianarb**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@ php:
   - 7
   - hhvm
 
+env:
+  - DOCTRINE_ORM="~2.4.0"
+  - DOCTRINE_ORM="~2.5.0"
```
Comment (Ocramius) [inline .travis.yml]: I'd rather bump the dependency to the ORM to 2.5 than introducing this sort of maintenance complexity.

Comment (gianarb) [inline .travis.yml]: do you increase dependency to doctrine/orm: ~2.5 and stop?

Comment (Ocramius) [inline .travis.yml]: Yes

**Comment by gianarb** [comment]
Fixed by #434

**Pull request closed without merging**